docs/publishers/cli.md

```diff
@@ -0,0 +1,65 @@
+# DPM: the datapackage manger CLI
+
+The dpm is a command-line tool aimed to help publishers to prepare and upload data to the Datapackage Registry. With dpm you will be able to:
+
+* Validate your data to ensure its quality
+* Upload datapackage
```

Comment:
Hm... Right now we call the command "publish" in dpmpy, and we may have more general issue with private packages. "To publish a private package" sounds a bit controversial to me. Calling the command "upload" may resolve this, but i personally like "publish" as it better describes the process for non-private data packages.